configure.ac

```diff
@@ -1,180 +0,0 @@
-dnl incantations, and it is easy to mess things up if you don't know what you
-dnl are doing.
-
-# Version numbers
-# ---------------
-# Name and version for pkg-config; increment when introducing breaking changes.
-# This version number is also the major version number.
-m4_define([_EOS_INSTRUMENTATION_MAJOR_VERSION_MACRO], [0])
-
-# Minor and micro versions: increment micro version when making a release. Minor
-# version is even for a stable release and odd for a development release.
-# When making any release, if the API changes, set the interface age to 0.
-m4_define([_EOS_INSTRUMENTATION_MINOR_VERSION_MACRO], [0])
-m4_define([_EOS_INSTRUMENTATION_MICRO_VERSION_MACRO], [0])
-
-# Full version, for use in AC_INIT
-m4_define([_EOS_INSTRUMENTATION_VERSION_MACRO],
-	[_EOS_INSTRUMENTATION_MAJOR_VERSION_MACRO._EOS_INSTRUMENTATION_MINOR_VERSION_MACRO._EOS_INSTRUMENTATION_MICRO_VERSION_MACRO])
-
-# Initialization
-# --------------
-# Initialize Autoconf: package name, version, bug report address, tarball name,
-# website
-AC_INIT([EOS Metrics Instrumentation Daemons], [_EOS_INSTRUMENTATION_VERSION_MACRO],
-	[], [eos-metrics-instrumentation], [http://endlessm.com])
-# Verify that the source directory can be found
-AC_CONFIG_SRCDIR([src/eos-metrics-instrumentation.c])
-# Initialize Automake: enable all warnings and do not insist on GNU standards.
-# no-portability suppresses warnings about syntax specific to GNU make.
-# parallel-tests specifies that we use the new parallel-running test harness.
-# Unlike serial-tests, this option is accepted by Automake 1.11.
-# subdir-objects ensures forward compatibility with Automake 2.0 and can be
-# removed when this behavior becomes the default.
-# tar-ustar is required because there may be files whose entire paths exceed
-# 99 characters.
-AM_INIT_AUTOMAKE([-Wall -Wno-portability foreign 1.11 parallel-tests
-    subdir-objects tar-ustar])
-# Avoid spewing garbage over the terminal ('make V=1' to see the garbage)
-AM_SILENT_RULES([yes])
-# Keep Autotools macros local to this source tree
-AC_CONFIG_MACRO_DIR([m4])
-
-AC_CACHE_SAVE
-
-# Variables to define
-# -------------------
-# Version numbers and macros
-EOS_INSTRUMENTATION_VERSION=_EOS_INSTRUMENTATION_VERSION_MACRO
-AC_SUBST(EOS_INSTRUMENTATION_VERSION)
-EOS_INSTRUMENTATION_MAJOR_VERSION=_EOS_INSTRUMENTATION_MAJOR_VERSION_MACRO
-AC_SUBST(EOS_INSTRUMENTATION_MAJOR_VERSION)
-AC_DEFINE([EOS_INSTRUMENTATION_MAJOR_VERSION], [_EOS_INSTRUMENTATION_MAJOR_VERSION_MACRO], [Major version])
-EOS_INSTRUMENTATION_MINOR_VERSION=_EOS_INSTRUMENTATION_MINOR_VERSION_MACRO
-EOS_INSTRUMENTATION_MICRO_VERSION=_EOS_INSTRUMENTATION_MICRO_VERSION_MACRO
-AC_DEFINE([EOS_INSTRUMENTATION_MINOR_VERSION], [_EOS_INSTRUMENTATION_MINOR_VERSION_MACRO], [Minor version])
-AC_DEFINE([EOS_INSTRUMENTATION_MICRO_VERSION], [_EOS_INSTRUMENTATION_MICRO_VERSION_MACRO], [Micro version])
-AC_DEFINE([EOS_INSTRUMENTATION_VERSION], [_EOS_INSTRUMENTATION_VERSION_MACRO], [Full version number])
```

Comment:
Note for posterity: these macros have all been dropped in the Meson version, presumably because they aren’t used anywhere in the code and the project installs no public headers.

Reply:
Yes, this is not a library, it's a pair of executables.